packages/volto/src/components/theme/Search/Search.jsx

```diff
@@ -112,6 +125,15 @@ class Search extends Component {
     const options = qs.parse(this.props.history.location.search);
     this.setState({ currentPage: 1 });
     options['use_site_search_settings'] = 1;
+
+    // Ensure default sort order for date fields
+    if (options.sort_on && !options.sort_order) {
+      options.sort_order = 'descending';
+    }
+
+    // Convert sort options for Plone compatibility
+    convertSortOptions(options);
```

Comment:
Now we have clear URL options and search options. You can create two variables, `urlOptions` and `searchOptions`. And create `searchOptions` based on `urlOptions`:

```javascript
const searchOptions = {
  use_site_search_settings: 1,
  b_size: this.defaultPageSize,
  ...urlOptions,
};
```

Here you can only pass `searchOptions`

```javascript
const apiOptions = convertSortOptions(searchOptions);
```